Paco/src/com/pacoapp/paco/net/ServerAddressBuilder.java

```diff
@@ -17,7 +17,7 @@ public static String createServerUrl(String serverAddress, String path) {
   }
 
   public static boolean isLocalDevelopmentServerAddress(String serverAddress) {
-    return serverAddress.contains("10.0.2.2");
+    return serverAddress.contains("10.0.2.2") || serverAddress.contains("127.0.0.1");
```

Comment:
Hi Bram, is this running a server on the phone? Or is this a new way to address the host machine attached to the phone in dev mode?